<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDialog1</class>
 <widget class="QDialog" name="AdminDialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1429</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>161</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadInfo">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>720</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Load Data</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>280</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource.qrc">:/img/vaccine.png</pixmap>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>641</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>341</width>
     <height>781</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_save_2">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_username">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_save_3">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_pw">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_save_4">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>CNIE</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_insur">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_save_5">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_fName">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_save_9">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Prenom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_lName">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_save_7">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_age">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButton_save_8">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Vaccin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_vaccine">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton_save_10">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Dose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_shot">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="pushButton_save_11">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_date">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="pushButton_save_12">
        <property name="styleSheet">
         <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_time">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_quit">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>720</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_retour">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>720</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource.qrc">:/img/HCW.png</pixmap>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>590</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resource.qrc">:/img/cc9976.png</pixmap>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>label_13</zorder>
  <zorder>label_12</zorder>
  <zorder>pushButton_loadInfo</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_delete</zorder>
  <zorder>pushButton_update</zorder>
  <zorder>label_11</zorder>
  <zorder>tableView</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>pushButton_quit</zorder>
  <zorder>pushButton_retour</zorder>
  <zorder>comboBox</zorder>
  <zorder>listView</zorder>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
